Tools/px_uploader.entrypoint

```diff
@@ -1,4 +1,14 @@
 #!/bin/sh -eu
 
-/bin/px_uploader.py --udp-addr=192.168.200.101 --udp-port=14541 --port=/dev/px4serial --baud-flightstack=57600,115200,1000000,2000000 --baud-bootloader=2000000 --use-protocol-splitter-format px4_fmu-v5_ssrc-*.px4 px4_fmu-v5x_ssrc-*.px4 ssrc_saluki-v1_default-*.px4
-/bin/px4-post-update-init -a :14540 $1
+aframe=""
+if [ ! -z "$AIRFRAME" ]; then
+	aframe="--airframe ${AIRFRAME}"
+fi
+
+mode=""
+if [ ! -z "$FLIGHT_MODE" ]; then
+	mode="--flightmode ${FLIGHT_MODE}"
```

Comment:
Good addition :muscle:

Reply:
Needed to remove "-u" the unset parameter check from the beginning of entrypoint script to allow run container even when those env variables does not exist.